batch.py

```diff
@@ -777,7 +778,7 @@ def manage_concurrency_for_job(job: hb.job.Job):
             manage_concurrency_for_job(prepare_input_job)
             copy_common_env(prepare_input_job)
             if gene_dict[gene]['plink_job'] is not None:
-                prepare_input_job.depends_on(gene_dict[gene]['plink_job'])
+                prepare_input_job.depends_on(dependencies_dict[gene]['plink_job'])
```

Comment:
GitHub doesn't let me make a multi-line suggestion here, but this should be:

```python
            dependency = dependencies_dict.get(gene)
            if dependency:
                prepare_input_job.depends_on(dependency)
```

Reply:
thanks!